web/sites/default/config/group.role.private-anonymous.yml

```diff
@@ -11,4 +11,5 @@ internal: true
 audience: anonymous
 group_type: private
 permissions_ui: true
-permissions: {  }
+permissions:
```

Comment:
For a restricted group (private), should an anonymous user have this permission? It probably doesn't matter at the eFiles file is likely public, so the use case would be pretty rare. I'm approving the PR, but we might follow up with a restriction to this role if we decide private groups should not have visible remote documents in things like search.

Reply:
Permission on Remote Document in Private group fixed.